### PR TITLE
Fix compile errors with latest ReactiveSwift

### PR DESCRIPTION
**Pull request by liscio**
This just gets the bridge compiling correctly with the latest ReactiveSwift / Xcode 9 stuff.

### CODE REVIEW

File: ReactiveObjCBridge/ObjectiveCBridging.swift [hunk 1]
```diff
@@ -495,6 +495,8 @@ extension DispatchTimeInterval {
 			return TimeInterval(UInt64(us) * NSEC_PER_USEC) / TimeInterval(NSEC_PER_SEC)
 		case let .nanoseconds(ns):
 			return TimeInterval(ns) / TimeInterval(NSEC_PER_SEC)
+		case let .never:
```
Comment (mdiep) [inline ReactiveObjCBridge/ObjectiveCBridging.swift]: This needs to be conditionally compiled (just like we did in ReactiveSwift) to maintain compatibility with Xcode 8.2/Swift 3.1.

**Comment by mdiep** [comment]
Closed in favor of #30.

**Comment by mdiep** [comment]
Actually, there are other bits here that we could use. 😅

File: Cartfile.private [hunk 2]
```diff
@@ -1,3 +1,3 @@
 github "jspahrsummers/xcconfigs" "3d9d996"
 github "Quick/Quick" ~> 1.1
-github "Quick/Nimble" "8116a83864ee78339798c3ef425c42f6ca6bf034"
+github "Quick/Nimble" ~> 6.1
```
Comment (ikesyo) [inline Cartfile.private]: Let's use `~> 7.0.1`

**Comment by sharplet** [comment]
I've been working on https://github.com/ReactiveCocoa/ReactiveObjCBridge/pull/51, but I just noticed this was here. Do we still need this PR?

**Comment by mdiep** [comment]
I don't think so?

**Comment by sharplet** [comment]
Closing this for now.

**Pull request closed without merging**